templates/sample/list_issues.html

```diff
@@ -0,0 +1,28 @@
+{% if issues|length > 0 %} 
+    <div id="no-more-tables">
+        <table class="col-md-12 table-bordered table-striped table-condensed cf">
+            <thead class="cf">
+                <tr>
+                    <th class="numeric">id</th>
+                    <th class="numeric">Title</th>
+                    <th class="numeric">Created By</th>
+                    <th class="numeric">Created At</th>
+                    <th class="numeric">Status</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for issue in issues %}
+                    <tr>
+                       <td data-title="id"><a href="{{ issue['html_url'] }}">{{ issue['number'] }}</a></td>
+                       <td data-title="Title">{{ issue['title'] }}</td>
```

Comment:
I'd merge this, so it looks more like it's on GH: {title} #{nr}, and then a link wrapping that.